src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  for (const value in robot) {
+    const isKey = robot[value];
```

Comment:
`isKey` refers to a boolean, let's give it another name

Reply:
+